pytorch_lightning/trainer/training_loop.py

```diff
@@ -83,6 +82,8 @@ def __init__(
         else:
             self.trainer.num_sanity_val_steps = num_sanity_val_steps
 
+        self.train_results = ResultCollection(True)
```

Comment:
are these the training results for one epoch, and after one epoch it will be cleared right?
I can't see where it gets updated?

Reply:
Gets updated by reference through the `trainer.result_collection` property